### PR TITLE
started implementation of qrcode generator, saving to database in bucket

### PR DESCRIPTION
**Pull request by kssampson**
-Not currently working, but shouldn't impact other code
-qr code should be stored as an image in a bucket called 'qrcodes' in supabase
-getting the qrcode buffer, but has supabase error 403 related to authorization violation of RLS policy

### CODE REVIEW

File: server/controllers/qrGeneratorController.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const { Reservations } = require('../db.js');
+const { supabase } = require('../db');
```
Comment (clee4037) [inline server/controllers/qrGeneratorController.js:2]: Lines 1 and 2 can be merged

**Review by clee4037 — APPROVED** [review]
Structure looks good! Not much to comment on. Looking forward to testing the QR code.

**Comment by kssampson** [comment]
-Removed unneeded require statements in qr generator files. 
-Removed the png file from tempImages directory

File: server/routes/qrGeneratorRouter.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+qrGeneratorRouter
+  .post('/:reservationId', async (req, res) => {
+
+    const reservationId = req.params.reservationId;
+    const urlToEndcode = req.body.yourUrl
+
+    const success = await generateAndStoreQRCode(urlToEndcode, reservationId);
+
+    if (success) {
+      res.json({ success: true, message: 'QR code generated and stored successfully.' });
```
Comment (kjmak14) [inline server/routes/qrGeneratorRouter.js:14]: maybe add a status code for both success and fail

**Review by kjmak14 — APPROVED** [review]
Seems good other than the exclusion of status codes in your routing. Nice work implementing qr code generation on the server side

**Pull request merged**